src/System Application/App/AI/src/Azure OpenAI/Chat Completion/AOAIPolicyParamsMapping.Codeunit.al

```diff
@@ -0,0 +1,29 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.AI;
+
+/// <summary>
+/// Interface for mapping AOAI policy parameters to AOAI policy enums.
+/// </summary>
+codeunit 7777 "AOAI Policy Params Mapping"
```

Comment:
Do we need to expose this codeunit and the enum? Can't it be an internal method within the AOAIPolicyParams codeunit? I guess our main recommendation will be to use the AOAIPolicyParam to define the type of validation, so we don't really have a need to expose the enum. The enum seems more like an internal concept not something that needs to be exposed.